src/Features/Blockcore.Features.Wallet/Api/Controllers/WalletRPCController.cs

```diff
@@ -120,15 +120,17 @@ public bool LockWallet()
 
         [ActionName("sendtoaddress")]
         [ActionDescription("Sends money to an address. Requires wallet to be unlocked using walletpassphrase.")]
-        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest)
+        public async Task<uint256> SendToAddressAsync(BitcoinAddress address, decimal amount, string commentTx, string commentDest, decimal? fee = null, bool isSegwit = false)
         {
-            WalletAccountReference account = this.GetWalletAccountReference();
+            decimal transactionFee = fee ?? this.FullNode.Network.MinTxFee;
```

Comment:
MintxFee is type long and defined as sats. The manual fee from method argument should be of same type and "decimal" (there are none on the mintxfee) position.

Reply:
Good catch!

Reply:
This is fixed @sondreb Please review.